src/main/java/frc/robot/Robot.java

```diff
@@ -12,28 +12,34 @@
 
 package frc.robot;
 
-import frc.bionic.UserInterfaceElement;
-import frc.bionic.swerve.AbstractDrivetrain;
-import edu.wpi.first.wpilibj.TimedRobot;
+import org.opencv.core.RotatedRect;
+
 import edu.wpi.first.networktables.NetworkTableEntry;
-import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
-import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import edu.wpi.first.wpilibj.TimedRobot;
+import edu.wpi.first.wpilibj.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.geometry.Translation2d;
 import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
 import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandScheduler;
+import frc.bionic.Conversion;
+import frc.bionic.TrajectoryFollow;
+import frc.bionic.TrajectoryUtils;
+import frc.bionic.UserInterfaceElement;
+import frc.bionic.swerve.AbstractDrivetrain;
```

Comment:
I don't see where many of these imports are used in this class. Maybe there's some cleanup to be done?